### ISSUE TITLE
Guice related exception: Could not load Finalizer in its own class loader.

### ISSUE THREAD

**Issue by evilDave** [roboguice/roboguice#325]
Getting this exception:

java.io.FileNotFoundException: com/google/inject/internal/util/$Finalizer.class

"Could not load Finalizer in its own class loader. Loading Finalizer in the current class loader instead. As a result, you will not be able to garbage collect this class loader. To support reclaiming this class loader, either resolve the underlying issue, or move Google Collections to your system class path."

Which is apparently fixed in guice 4.0
Are there any plans on updating the fork of guice to 4.0? Or incorporating the changes that fix this warning? (Seems like they stopped using FinalizableReferenceQueue)

**Comment by stephanenicolas** [comment]
Yeah, the next release candidate has upgraded to Guice 4, but we couldn't
push it to GH due to a size constraint, we are gonna fix that.

S.

Le jeu. 29 oct. 2015 19:37, David K Clark <notifications@github.com> a
écrit :
